cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -336,6 +343,8 @@ private String rebalanceStateConditionType(KafkaRebalanceStatus status) {
                                    KafkaRebalanceState currentState, KafkaRebalanceAnnotation rebalanceAnnotation) {
 
         log.info("{}: Rebalance action from state [{}]", reconciliation, currentState);
+        log.info("" + kafkaRebalance.getMetadata().getNamespace());
+        log.info("" + reconciliation.namespace());
```

Comment:
This looks like some debug logging ... maybe it can be removed?

Reply:
Sorry. I have removed them now